app/sidekiq/hca/log_email_diff_job.rb

```diff
@@ -28,5 +36,29 @@ def perform(in_progress_form_id, user_uuid)
       )
       $redis.set(redis_key, 't')
     end
+
+    def log_different_email(in_progress_form_id, user_uuid)
+      return if InProgressEmailMatchLog.exists?(user_uuid:, in_progress_form_id:)
```

Comment:
I don't think so, because of the unique constraint [in this PR](https://github.com/department-of-veterans-affairs/vets-api/pull/19397). If two concurrent jobs both returned true for `exists?`, one would fail when trying the create since one of them would successfully create the record and the index would throw the exception (which would just kill the job). I think that should handle it?

Reply:
Yeah, your right. Poor wording on my part. The create will fail, and since it's at the end of the job and there are no retries for this job, it wouldn't cause any other issues. But I think we should be okay as far as race conditions, though. I appreciate the call out on it! 👍